src/components/ComposerQuill/composer.js

```diff
@@ -177,6 +177,29 @@ class Composer extends React.Component {
           const contents = buildContents(draft.value, mentions?.participants?.current);
 
           this.quill.setContents(contents);
+          let text = this.quill.getText();
+
+        // modifying list in case of space switch
+          while (text.match(/\* /)) {
```

Comment:
This while loop is concerning to me inside of the component did update. Especially based on a regex.

Reply:
I moved this loop inside separate function which call from emitter only. It doesn't bother componentDidUpdate now.